### ISSUE TITLE
How to load my dataset as an example from keras?

### ISSUE THREAD

**Issue by mlitvinov** [keras-team/keras#13604]
I use the code from [Keras example](https://keras.io/examples/cifar10_cnn/) There, the data is loaded as a line 

> (x_train, y_train), (x_test, y_test) = cifar10.load_data() 

and everything works correctly, but when I try to upload my pictures using

How to load your data into this example? I tried to find implementations, everything is based on open datasets that load the same way as in the Keras example.

**Comment by am1tyadav** [comment]
If I understand the question correctly, you are trying to use your own images instead of a data set that comes pre-packaged with keras. The `cifar10.load_data()` function returns two tuples of numpy arrays for training and test set. So, if you are using your own images, you will have to do this type of processing and convert your images. For example, something simple like this would do the job for a single image:
```
import numpy as np
from PIL import Image

img = Image.open(path_to_image)
img = img.resize((width, height), Image.ANTIALIAS)
img_as_ndarray = np.array(img)
```
Keep in mind that the width and height need to be set to what your neural network expects at input for each example, so something like (32, 32, 3) for each example. Once you have a collection of all your images converted to multi dimensional arrays, you can split them into training and test sets.
